atom/browser/native_window_mac.mm

```diff
@@ -240,6 +240,12 @@ - (BOOL)windowShouldClose:(id)window {
   return NO;
 }
 
+- (NSRect)window:(NSWindow *)window willPositionSheet:(NSWindow *)sheet usingRect:(NSRect)rect {
+  rect.origin.y = window.contentView.frame.size.height - shell_->GetSheetOffset();
```

Comment:
Got a compilation error in CI machine:

```
../../atom/browser/native_window_mac.mm:244:38: error: property 'frame' not found on object of type 'id'
  rect.origin.y = window.contentView.frame.size.height - shell_->GetSheetOffset();
```

We can probably try `NSView* contentView = window.contentView`, since the type of `window.contentView` is `id`.